test/generator/msan_aottest.cpp

```diff
@@ -19,25 +19,87 @@ int main(int argc, char **argv) {
 using namespace std;
 using namespace Halide::Runtime;
 
+enum {
+    kAnnotate_bounds_inference_buffer,
```

Comment:
elsewhere for enums we just use ThisSortOfThing

Reply:
done